### PR TITLE
Score capture

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 1]
```diff
@@ -99,37 +103,47 @@ export default class search extends Component {
 
   Validate() {
     let Valid = false;
-    if( parseInt(this.state.score) < 1 || this.state.score === null || this.state.score === "" || (this.state.score % 1) !== 0) 
-    {
+    if (parseInt(this.state.score) < 1 || this.state.score === null 
+    || this.state.score === "" || (this.state.score % 1) !== 0) {
```
Comment (ash-rabbit) [inline skietbaan_fe/src/components/ScoreCapture.js]: this doesn't look right, is score a string or int?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/skietbaan_fe/src/components/ScoreCapture.css.orig
```
Comment (FredJEhlers) [inline .gitignore]: Why that one specific and not all `*.orig`

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 3]
```diff
@@ -279,94 +295,146 @@ export default class search extends Component {
...
     }
 
     return (
       <div className="position-relative">
-        <div className={this.state.showCamera && !this.state.ImageTaken ? "back-white page-content no-padding" :
-          this.state.ImageTaken ? "opacity-85 page-content no-padding" : "page-content no-padding"}>
-          <div className={this.state.showCamera && !this.state.ImageTaken ? "hidden" : ""}>>
-        <div className="label-score">
-              <label>Type in score</label>
+        <div className={this.state.showCamera || this.state.ImageTaken ? "" : "white-border"}></div>
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ScoreCapture.js]: Extract this comparison as it is done three times

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 4]
```diff
@@ -279,94 +295,146 @@ export default class search extends Component {
...
+                <img src={require('../components/assets/scoreCapture.png')}
+                  id="btnScoreCapture" className="btnScoreCapture"
+                  onClick={() => this.CameraClicked()} alt=''></img>
+              </div>
+              <label className="labelIcon">Capture score</label>
+            </div>
+            <div className={(this.state.showCamera && !this.state.ImageTaken) 
+              || this.state.ImageTaken || !this.state.scoreEntered ? "hidden" : "submit-button-elements"}>
+              <div className="button-hover ">
+                <img src={require('../components/assets/submitScore.png')} onClick={() => this.GetLocation()}
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ScoreCapture.js]: You have used this path `../components/assets/submitScore.png` 3 times or more. 
please make use of a constant

Comment (FredJEhlers) [reply skietbaan_fe/src/components/ScoreCapture.js]: You don't have to right but let us say the image changes to a different one, you have to find each occurrence or you could change it once in the constants

File: skietbaan_fe/src/components/ScoreCapture.js (hunk 1 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: ok removed it

File: .gitignore (hunk 2 above)
Comment (ilzeLourens) [reply .gitignore]: Because thats the only one. I think

File: skietbaan_fe/src/components/ScoreCapture.js (hunk 4 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: The image is never being changed to a different one its just being hidden. I tried making the image path a constant but its not working

File: .gitignore (hunk 2 above)
Comment (ash-rabbit) [reply .gitignore]: He meant that add for every .orig file.

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 5]
```diff
@@ -99,37 +103,47 @@ export default class search extends Component {
 
   Validate() {
     let Valid = false;
-    if( parseInt(this.state.score) < 1 || this.state.score === null || this.state.score === "" || (this.state.score % 1) !== 0) 
-    {
+    if (parseInt(this.state.score) < 1 || this.state.score === null 
+    || (this.state.score % 1) !== 0) {
```
Comment (ash-rabbit) [inline skietbaan_fe/src/components/ScoreCapture.js]: I mean if score is string, convert it so you can check for mod validation in your case `this.state.score % 1`
as well as check for empty string like you were doing it previously

Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: I dont allow for string inputs

Comment (ash-rabbit) [reply skietbaan_fe/src/components/ScoreCapture.js]: Is Score string or Int?

Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: its an int . The input type is set to number

File: skietbaan_fe/src/components/ScoreCapture.js (hunk 3 above)
Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: done

**Pull request merged**